fontc_crater/src/ci/html.rs

```diff
@@ -91,7 +91,22 @@ fn make_html(
             }
         }
     }
-    .into_string()
+    .into_string();
+    tidy_html(&raw_html)
+}
+
+fn tidy_html(raw_html: &str) -> Result<String, Error> {
+    let opts = tidier::FormatOptions {
+        // indent with tabs to reduce file size.
+        // the '4' option is suggested by the docs; does not mean 4 tabs each indent?
+        indent: tidier::Indent {
+            size: 4,
+            tabs: true,
```

Comment:
I cannot endorse use of tabs or 4 space indents, let the great war resume :D

Reply:
not my preference either, but compared to 4 spaces this reduces file size from 1.9MB to 900kb 😒